### PR TITLE
Add Jerk, L^1 * T^-3

### PR DESCRIPTION
**Pull request by nicodemus26**
Useful for robotics and motion planning.

Also added some common CNC units to velocity and acceleration.

### CODE REVIEW

File: src/si/jerk.rs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            test::<l::centimeter, j::centimeter_per_second_cubed>();
+            test::<l::millimeter, j::millimeter_per_second_cubed>();
+            test::<l::micrometer, j::micrometer_per_second_cubed>();
+            test::<l::nanometer, j::nanometer_per_second_cubed>();
+            test::<l::picometer, j::picometer_per_second_cubed>();
+            test::<l::femtometer, j::femtometer_per_second_cubed>();
+            test::<l::attometer, j::attometer_per_second_cubed>();
+            test::<l::zeptometer, j::zeptometer_per_second_cubed>();
+            test::<l::yoctometer, j::yoctometer_per_second_cubed>();
+
```
[the remark this answers is not in the document]
Comment (nicodemus26) [reply src/si/jerk.rs]: Galileo snuck in from Acceleration, I've removed it from Jerk.

I've parameterized time for all of these tests and added the tests you requested.

**Comment by nicodemus26** [comment]
If approved, could you cut a minor or patch release that I may pin in my project?

Thanks kindly!

File: src/si/jerk.rs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            test::<l::zeptometer, t::second, j::zeptometer_per_second_cubed>();
+            test::<l::yoctometer, t::second, j::yoctometer_per_second_cubed>();
+            test::<l::foot, t::second, j::foot_per_second_cubed>();
+            test::<l::inch, t::second, j::inch_per_second_cubed>();
+            test::<l::kilometer, t::minute, j::kilometer_per_minute_cubed>();
+
+            fn test<L: l::Conversion<V>, T: t::Conversion<V>, J: j::Conversion<V>>() {
+                Test::assert_eq(
+                    &Jerk::new::<J>(V::one()),
+                    &(Length::new::<L>(V::one()) /
```
Comment (iliekturtles) [inline src/si/jerk.rs]: Start next line with `/` operator. Do the same for the `*` operators. Tab in the contents of the inner-most `()`s where the time^3 factor is calculated.

```rust
            fn test<L: l::Conversion<V>, T: t::Conversion<V>, J: j::Conversion<V>>() {
                Test::assert_eq(
                    &Jerk::new::<J>(V::one()),
                    &(Length::new::<L>(V::one())
                        / (Time::new::<T>(V::one())
                            * Time::new::<T>(V::one())
                            * Time::new::<T>(V::one()))));
            }
```

**Comment by iliekturtles** [comment]
Thanks again for the PR! There are a couple minor changes I'm going to include in another commit shortly rather than have you do it. My plan is do some more review of the `Torque` PR tomorrow and release a new version to crates.io either way.

**Comment by iliekturtles** [comment]
[v0.22.2](https://github.com/iliekturtles/uom/releases/tag/v0.22.2) released!

**Pull request merged**